batchlib/analysis/cell_level_analysis.py

```diff
@@ -685,20 +710,31 @@ def load_image_outliers(self, input_files):
                         for ii in range(len(table))}
         return outlier_dict
 
-    def load_cell_outliers(self, input_file):
-        with open_file(input_file, 'r') as f:
-            if not self.has_table(f, self.cell_outlier_table):
-                logger.warning(f"{self.name}: load_cell_outliers: did not find a cell outlier table")
```

Comment:
Why are we getting rid of this? Or did this just move somewhere else?

Reply:
I just refactored all of this to the new free function `load_cell_outlier_dict`, that is also used in `WellLevelQC`. The overall functionality has stayed the same.

Reply:
Ah I see, thank you for the explanation.